File: src/test/java/stringcalculator/Number.java
package stringcalculator;

public class Number {
    private final Integer value;
    private static final int ZERO = 0;

    public Number(Integer input){
        if (input < ZERO) throw new RuntimeException();
        this.value = input;
    }

    public Integer getValue() {
        return value;
    }
}

File: src/test/java/StringCalculator/Number.kt
package StringCalculator

object Number {
    fun create(text: String): Int {

        val number = text.toIntOrNull() ?: 0

        require(number >= 0) {
            throw RuntimeException("음수 값이 포함되어 있습니다.")
        }

        return number
    }
}